hack/export-certs.sh

```diff
@@ -22,8 +22,8 @@ if [[ -z "${CERT_DIR}" ]]; then
     exit 1
 fi
 
-export CURL_CA_BUNDLE="${CERT_DIR}/root.crt"
 export CURL_CERT="${CERT_DIR}/cert.crt"
 export CURL_KEY="${CERT_DIR}/key.key"
+export CURL_CA_BUNDLE="${CERT_DIR}/../ca/cert.crt"
```

Comment:
check if the file exists or if CURL_CA_BUNDLE is already pointing at a valid file